.travis.yml

```diff
@@ -4,7 +4,10 @@ jdk:
   - oraclejdk8
   - openjdk7
 
-
+  
+before_install:
+        - sudo apt-get install libboost-dev libboost-test-dev
```

Comment:
this will not work on windows. or on any non deb base system... isnt there a better way of handling these...